lms/djangoapps/lti_provider/tests/test_management.py

```diff
@@ -0,0 +1,99 @@
+"""
+Test lti_provider management commands.
```

Comment:
Instead of having one test file for all management commands, we typically have a separate test file for each management command.  You can rename this file so it's specific to testing the specific command and move it to a new test subdirectory within the management directory.